addons/resource.language.en_gb/resources/strings.po

```diff
@@ -9817,15 +9820,15 @@ msgid "Login screen"
 msgstr ""
 
 msgctxt "#20097"
-msgid "Fetching album info"
+msgid "Fetching album information"
 msgstr ""
 
 msgctxt "#20098"
-msgid "Fetching info for album"
+msgid "Fetching information for album"
 msgstr ""
 
 msgctxt "#20099"
-msgid "Can't rip CD or track while playing from CD"
+msgid "Can not rip a CD or track while playing from the CD"
```

Comment:
This reads odd, how about "A CD or track can't be ripped while being played". Also I personally prefer "can't" over "can not", but I don't know how formal the wording is in other software. @jjd-uk ?